src/NServiceBus.Metrics.PerformanceCounters/PerformanceCounterUpdater.cs

```diff
@@ -28,7 +29,8 @@ public void Update(string payload)
         foreach (var timer in timers)
         {
             var performanceCounterInstance = cache.Get(new CounterInstanceName(timer.Name, context));
-            performanceCounterInstance.RawValue = timer.TotalTime;
+            
+            performanceCounterInstance.RawValue = timer.Histogram.LastValue / 1000;
```

Comment:
Well, had a discussion with Ramon which resulted in this issue : https://github.com/Particular/NServiceBus.Metrics/issues/30
Which is in wrong repo, but as a result I thought we could create two new counters. One for `CriticalTime` and one for `ProcessingTime` which do report in ms and also use the correct counter for PerfMon.
I'll think of something to report on the division by 1000 in the meantime.